qubes/vm/qubesvm.py

```diff
@@ -1365,7 +1381,7 @@ async def start(
                     f"Qube start is prohibited. Rationale: {prohibit_rationale}"
                 )
 
-            self.log.info("Starting {}".format(self.name))
+            self.log.info("Starting qube")
```

Comment:
AFAIR there are cases where logger name is not easily accessible, so better keep the name, at least in this message.